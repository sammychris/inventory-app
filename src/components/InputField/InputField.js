import React from 'react';
import styled from 'styled-components';


const InputField = ({value, name, event, placeholder="", type="", required=false}) => {
    return (
        <Input value={value} name={name} onChange={event} placeholder={placeholder} type={type} required={required} />
    );
  }


export default InputField;

const Input = styled.input`
  width: 500px;
  height: 30px;
  font-size: 18px;
  margin: 5px 0;
`;

