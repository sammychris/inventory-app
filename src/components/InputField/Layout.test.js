import { shallow } from "enzyme";
import InputField from "./InputField";

const wrapper = shallow(<InputField />);

test("expect to render a InputField", () => {
  expect(wrapper.length).toEqual(1);
});

test("expect to make a snap shot to InputField component", () => {
  expect(wrapper).toMatchSnapshot();
});