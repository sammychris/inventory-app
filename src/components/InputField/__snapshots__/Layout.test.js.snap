// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to make a snap shot to InputField component 1`] = `ShallowWrapper {}`;
